### PR TITLE
Recommendation frontend

### PR DESCRIPTION
**Pull request by phlpphcknr**
- styling
- logo added
- set up requirement to have at least one tag to set to get or set a recommendation

### CODE REVIEW

File: frontend/src/components/AppHeader.js [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
-    margin: 0px;
-    text-align: center;
-    font-size: 0.6em;
-    color: black;
+  img {
+    display: block;
+    max-width: 70%;
+    width: auto;
+    height: auto;
+    margin: 4% 15% 2% 15%;
```
Comment (christophersiem) [inline frontend/src/components/AppHeader.js:25]:
```suggestion
    margin: 4% 15% 2%;
```

File: frontend/src/components/ArtistRecommendation.js [hunk 2]
```diff
@@ -74,12 +91,21 @@ const ArtistRecommender = styled.section`
   background: var(--primary-color);
   box-shadow: 0px 2px 4px #333;
 
-  button{
-    margin: 20px 0px 0px 0px;
+  .warning{
+    font-size: 16px;
+    text-align: center;
+    margin: 15px 30px 0px 30px;
```
Comment (christophersiem) [inline frontend/src/components/ArtistRecommendation.js:97]:
```suggestion
    margin: 15px 30px 0px;
```

File: frontend/src/components/GetArtistRecommendation.js [hunk 3]
```diff
@@ -64,13 +70,19 @@ const ArtistRecommender = styled.section`
   margin: 0px 20px;
   background: var(--primary-color);
   box-shadow: 0px 2px 4px #333;
+
+  .warning{
+    font-size: 16px;
+    text-align: center;
+    margin: 15px 30px 0px 30px;
```
Comment (christophersiem) [inline frontend/src/components/GetArtistRecommendation.js:77]:
```suggestion
    margin: 15px 30px 0px;
```

File: frontend/src/pages/ArtistOverview.js [hunk 4]
```diff
@@ -84,3 +90,11 @@ const Overview = styled.div`
   }
 
 `
+
+const Loading =styled.section`
```
Comment (christophersiem) [inline frontend/src/pages/ArtistOverview.js:94]: maybe create reusable Loading component? This should save you some lines of code

**Review by christophersiem — APPROVED** [review]
lgtm, left some comments :)

File: frontend/src/components/AppHeader.js (hunk 1 above)
Comment (phlpphcknr) [reply frontend/src/components/AppHeader.js:25]: thx. done

File: frontend/src/components/ArtistRecommendation.js (hunk 2 above)
Comment (phlpphcknr) [reply frontend/src/components/ArtistRecommendation.js:97]: thx. done

File: frontend/src/components/GetArtistRecommendation.js (hunk 3 above)
Comment (phlpphcknr) [reply frontend/src/components/GetArtistRecommendation.js:77]: thx. done

File: frontend/src/pages/ArtistOverview.js (hunk 4 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistOverview.js:94]: I will have a look into this ...

**Pull request merged**